jekyll/_cci2/jira-plugin.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: classic-docs
+title: "Connecting JIRA with CircleCI"
+categories: [how-to]
+description: "Connecting JIRA with CircleCI"
+---
+
+This document describes how you can connect JIRA to your CircleCI builds. With
+the CircleCI JIRA plugin, you can create JIRA tickets directly from your Jobs
+page, allowing you assign tasks and fixes based on the status of your build.
```

Comment:
allowing you assign -> allowing you to assign
should this be "status of your job" instead of "build"?

Reply:
Job please.

Reply:
ok, will switch back!